lab_6_pipeline/pipeline.py

```diff
@@ -165,23 +348,75 @@ def __init__(self, corpus_manager: CorpusManager):
         """
         Initializes MorphologicalAnalysisPipeline
         """
+        super().__init__(corpus_manager)
+        self._corpus = corpus_manager
+        self._backup_analyzer = pymorphy2.MorphAnalyzer()
+        self._path = Path(__file__).parent/'data'/'opencorpora_tags_mapping.json'
+        self._backup_tag_converter = OpenCorporaTagConverter(self._path)
 
     def _process(self, text: str) -> List[ConlluSentence]:
         """
         Returns the text representation as the list of ConlluSentence
         """
+        sentences = split_by_sentence(text)
```

Comment:
suggest a way to eliminate duplication between pipeline and advanced pipeline. what is the only difference between them?